src/components/Modal/Modal.scss

```diff
@@ -0,0 +1,26 @@
+.modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width:100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.6);
+  }
+  
+  .modal-main {
+    position:fixed;
+    background: white;
+    width: 80%;
+    height: auto;
+    top:50%;
+    left:50%;
+    transform: translate(-50%,-50%);
+  }
```

Comment:
we have sass enabled build, you probably wanna write in a format like 
`.modal{
 &.-main {
  ...
  }
}`

Reply:
not seen much needed